### PR TITLE
chore: Substrate Asset Transfer example

### PR DESCRIPTION
**Pull request by FSM1**
## Description
Added basic Substrate Asset example

## Related Issue Or Context

Closes: #243 

Has This Been Tested? Testing details.
To test this example:
1. Build the SDK `yarn sdk:build`
2. `cd examples/substrate-asset-transfer`
3. Install example dependencies `yarn`
4. Run the example `yarn transfer`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation

## Checklist:
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [x] I have updated the documentation locally and in chainbridge-docs.
- [x] I have added tests to cover my changes.
- [x] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: examples/substrate-asset-transfer/src/transfer.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+  const api = await ApiPromise.create({ provider: wsProvider });
+
+  const assetTransfer = new SubstrateAssetTransfer();
+
+  await assetTransfer.init(
+    api,
+    Environment.TESTNET,
+    SubstrateParachain.ROCOCO_PHALA
```
Comment (mpetrun5) [inline examples/substrate-asset-transfer/src/transfer.ts:35]: Can we make this a required param please and remove the default `LOCAL` so nobody makes that mistake please.

File: packages/sdk/src/chains/Substrate/assetTransfer.ts [hunk 2]
```diff
@@ -48,8 +48,8 @@ export class SubstrateAssetTransfer {
 
   public async init(
     apiPromise: ApiPromise,
-    environment: Environment = Environment.MAINNET,
-    parachainId: SubstrateParachain = SubstrateParachain.LOCAL,
+    environment: Environment,
```
Comment (mpetrun5) [inline packages/sdk/src/chains/Substrate/assetTransfer.ts:51]: I would leave this as default and move it below parachainId

File: packages/sdk/src/localConfig.ts [hunk 3]
```diff
@@ -63,7 +63,7 @@ export const localConfig: RawConfig = {
           type: ResourceType.FUNGIBLE,
           assetName: 'USDC',
           assetId: 2000,
-          xsmMultiAssetId: {
+          xcmMultiAssetId: {
```
Comment (mpetrun5) [inline packages/sdk/src/localConfig.ts:66]: Bit out of scope but can you remove my comments from this file which are a bit random.

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

File: packages/sdk/src/chains/Substrate/assetTransfer.ts (hunk 2 above)
Comment (FSM1) [reply packages/sdk/src/chains/Substrate/assetTransfer.ts:51]: Done

File: examples/substrate-asset-transfer/src/transfer.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  const resources = assetTransfer.config.getDomainResources();
+  const selectedResource = resources.find(
+    (resource) => resource.resourceId == RESOURCE_ID
+  );
+  if (!selectedResource) {
+    throw new Error("Resource not found");
+  }
+  const rococo = domains.find((domain) => domain.chainId == ROCOCO_CHAIN_ID);
+  if (!rococo) {
+    throw new Error("Network goerli not supported");
```
Comment (mpetrun5) [inline examples/substrate-asset-transfer/src/transfer.ts:47]: Update to match networks

Comment (FSM1) [reply examples/substrate-asset-transfer/src/transfer.ts:47]: Fixed

File: packages/sdk/src/localConfig.ts (hunk 3 above)
Comment (FSM1) [reply packages/sdk/src/localConfig.ts:66]: Done

File: examples/substrate-asset-transfer/README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
+## Sygma SDK ERC20 Example
```
[the remark this answers is not in the document]
Comment (FSM1) [reply examples/substrate-asset-transfer/README.md:1]: Done

**Comment by wainola** [comment]
@FSM1 just minor comment: can we add some finally to exit the script execution?

**Review by wainola — APPROVED** [review]

**Pull request merged**